src/pages/partners/payment-agent/_who-can-apply.tsx

```diff
@@ -70,6 +70,17 @@ const ButtonWrapper = styled(CenteredSection)`
     display: flex;
     justify-content: center;
 `
+const StyledListHeader = styled(Header)`
+    width: 100%;
+    max-width: 42.8rem;
+    margin-top: 4rem;
+
+    @media ${device.tabletL} {
+        font-size: 14px;
+        line-height: 20px;
+    }
+`
```

Comment:
Widths, max widths, margins should be set on its parent wrapper instead. Font size and line height adjust for responsive shouldn't be needed if the correct `type` is used.